clippy_lints/src/methods/mod.rs

```diff
@@ -3752,6 +3753,29 @@ declare_clippy_lint! {
     "using `Option.map_or(Err(_), Ok)`, which is more succinctly expressed as `Option.ok_or(_)`"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for iterators of `Result`s using ``.filter(Result::is_ok).map(Result::unwrap)` that may
+    /// be replaced with a `.flatten()` call.
+    ///
+    /// ### Why is this bad?
+    /// `Result` implements `IntoIterator<Item = T>`. This means that `Result` can be collected
```

Comment:
```suggestion
    /// `Result` implements `IntoIterator<Item = T>`. This means that `Result` can be flattened
```

Reply:
The latest push seems to have lost this change

Reply:
Must have been lost on rebasing 🙈🤦🏼‍♀️